src/main/java/io/jenkins/plugins/opentelemetry/job/MonitoringRunListener.java

```diff
@@ -104,42 +115,60 @@ public void postConstruct() {
 
         // METRICS
         activeRunGauge = new AtomicInteger();
+
+        runDurationHistogram = meter.histogramBuilder("ci.pipeline.run.duration")
```

Comment:
How are the buckets for the histogram defined?

Reply:
```
        runDurationHistogram = meter.histogramBuilder("ci.pipeline.run.duration")
        // TODO clarify histogram buckets.
```
Great catch Christophe, I forgot to add a TODO to clarify buckets